File: apps/s-2-admin/src/api/account/AccountUpdateInput.ts
import { UserWhereUniqueInput } from "../user/UserWhereUniqueInput";

export type AccountUpdateInput = {
  typeField?: string;
  provider?: string;
  providerAccountId?: string;
  refreshToken?: string | null;
  accessToken?: string | null;
  expiresAt?: number | null;
  tokenType?: string | null;
  scope?: string | null;
  idToken?: string | null;
  sessionState?: string | null;
  user?: UserWhereUniqueInput | null;
};

File: apps/cactus-demo-dotnet/src/APIs/Account/Dtos/AccountUpdateInput.cs
namespace CactusDemoDotnet.APIs.Dtos;

public class AccountUpdateInput
{
    public string? Id { get; set; }

    public string? TypeField { get; set; }

    public string? Provider { get; set; }

    public string? ProviderAccountId { get; set; }

    public string? RefreshToken { get; set; }

    public string? AccessToken { get; set; }

    public int? ExpiresAt { get; set; }

    public string? TokenType { get; set; }

    public string? Scope { get; set; }

    public string? IdToken { get; set; }

    public string? SessionState { get; set; }

    public int? User { get; set; }
}
